src/nix/command.cc

```diff
@@ -31,6 +31,30 @@ void StoreCommand::run()
     run(getStore());
 }
 
+EvalCommand::EvalCommand()
+{
+    addFlag({
+        .longName = "start-repl-on-eval-errors",
```

Comment:
```suggestion
        .longName = "debugger",
```

This is going to be used quite often so having a short long name is nice :)